deploy/cluster_role.yaml

```diff
@@ -0,0 +1,25 @@
+# Define Role that allows operations on K8s pods/deployments
+#  in "default" namespace
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: openebs-ndm
+  namespace: default
```

Comment:
can `namespace` be removed from all the metadata files? By removing it, can the user run the yamls in a custom namespace using the following commands:

```
kubectl create namespace < custom namespace > 
kubectl apply -n < custom namespace > -f < yaml file >
```

If namespace is not specified while applying yaml, will it work in default namespace.

Reply:
> By removing it, can the user run the yamls in a custom namespace 

yes.

> If namespace is not specified while applying yaml, will it work in default namespace.

yes.

The changes has been done.